pyaerocom/colocation/colocation_setup.py

```diff
@@ -492,6 +496,13 @@ def validate_obs_config(cls, v: PyaroConfig):
                 cls.obs_id = v.name
         return v
 
+    @cached_property
+    def basedir_logfiles(self):
+        p = Path(self.basedir_coldata) / "logfiles"
+        if not p.exists():
+            p.mkdir(parents=True, exist_ok=True)
```

Comment:
I see no need for the `if`, the `exist_ok=True` parameter does the checking.
You can simple write:
``` python
        p.mkdir(parents=True, exist_ok=True)
```